src/request.js

```diff
@@ -175,11 +175,13 @@ module.exports = class Request extends EventEmitter {
   validateParameters() {
     for (let i = 0, len = this.parameters.length; i < len; i++) {
       const parameter = this.parameters[i];
-      const value = parameter.type.validate(parameter.value);
+      const value = parameter.type.validate(parameter.value, parameter.length, parameter.precision, parameter.scale);
```

Comment:
Precision and scale are never actually used the validate(), let's remove them.

Reply:
Yeah, that's a good point. They *should* be used for `numeric` and `decimal` (and maybe for some of the date types?). I'll take another look! 👍

Reply:
Done in 0968cb4.